templates/games/chapter.html

```diff
@@ -0,0 +1,182 @@
+{% extends 'base.html' %}
+{% block body %}
+
+<div class="quiz_wrapper columns mb-6">
+	{% include 'chapter_menu.html' %}
+		<div class="quiz column is-four-fifths p-6">
+			<div class="section">
+				<p class="is-size-3 has-text-weight-bold mb-3">Level {{chapter.level_id}} / Chapter {{chapter.order_id}}: {{chapter.name}}</p>
+				<!--<div><a href="slides.html">Slides</a></div> -->
+				{% if chapter.url %}
+				<figure class="image is-16by9">
+					<iframe class="has-ratio" width="600" height="450" src="https://www.youtube.com/embed/{{chapter.url}}" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+				</figure>
+				{% endif %}
+			</div>
+			<hr>
+			{% if questions %}
+				<div id="quiz"></div>
+			{% endif %}
+			{% if chapter.id == 6 %}
```

Comment:
Why are the id starting at 6 here?

Reply:
6 is the chapter id for level 2 chapter 1 in the database, since chapters' pages for level 2 didn't follow the "video then quiz" structure, then I have to set the content manually.

Reply:
Can we restart the id numbers for level 2? So it is not breaking things if we change the number of chapters in level 1

Reply:
yes, the chapter also has an 'order_id' attribute which starts from 1 for its level. I can use it instead, will fix it soon.